### PR TITLE
Flag to get either the accepted changes or the original text

### PR DESCRIPTION
**Pull request by deayalar**

### CODE REVIEW

File: docx2python/docx_text.py [hunk 1]
```diff
@@ -261,10 +264,17 @@ def branches(tree: EtreeElement) -> None:
...
+            text = tree.text if tree.text is not None else ""
+            tables.add_text_into_open_run(text)
+
+        elif tree.tag == Tags.INSERTED:
+            is_inserted = True
 
         if do_descend:
             for branch in tree:
-                branches(branch)
+                branches(branch, is_inserted or False)
```
Comment (sebassdc) [inline docx2python/docx_text.py:277]: You don't need the or False in here

**Review by sebassdc — CHANGES_REQUESTED** [review]

**Comment by bholligan** [comment]
Can you add a little more clarity on how you plan on using the "only_deleted" argument to empower the end goal of notifying the user which sections were deleted and then using the playbook to tell them whether that's acceptable?

**Comment by bholligan** [comment]
What should happen when the following combination of flags occurs?
- is_deleted (True) & accepted_changes (False)

File: docx2python/docx_text.py (hunk 1 above)
Comment (deayalar) [reply docx2python/docx_text.py]: done!

**Comment by deayalar** [comment]
@bholligan 
I changed the flags by a parameter that can take three values "accepted", "original" or "deleted"

**Review by bholligan — APPROVED** [review]
lgtm

**Pull request merged**